### PR TITLE
Fix indentation on new.html.erb

### LINKED ISSUE #11077: Fix indentation in new.html.erb
### 💡Format this Issue Into a First Timers Issue
This has been marked as a good candidate for becoming a [first-timers-only issue](https://code.publiclab.org/#r=all) like these, meaning that it's **simple, self-contained**, and with some extra formatting✍️, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider **reformatting it as a first-timers-only issue**, and then ping @publiclab/reviewers to get it labelled🏷. Or, if this is not your first time, **try to solve it yourself**🧑‍💻!

****
### todo:
Fix indentation  of this file [new.html.erb](https://github.com/publiclab/plots2/edit/main/app/views/csvfiles/new.html.erb) in the `plots2` repository (press the little pen Icon). Indentation should be 2 spaces deep.

### PR DESCRIPTION
**Pull request by lanefoxwood**
I believe the task was to set the indentation to 2 spaces throughout the document. I attempted to do this...and after combing the whole doc, I could only see about 20ish lines that needed changing. I hope that these changes are accurate. Thank you for reviewing!

Fixes #11077 <a href="https://github.com/publiclab/plots2/issues/11077#issue-1209458348">original issue, comment</a> 

screenshot of proposed changes: 
Screen Shot 2022-09-13 at 15 26 45

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] PR is descriptively titled 📑 and links the original issue above 🔗
* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] code is in uniquely-named feature branch and has no merge conflicts 📁
* [X] screenshots/GIFs are attached 📎 in case of UI updation
* [X] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by lanefoxwood** [comment]
@publiclab/reviewers thank you for stepping in to help!

File: app/views/csvfiles/new.html.erb [hunk 1]
```diff
@@ -10,27 +10,27 @@
...
-    var value = '<%= current_user %>';
-    <% if current_user %>
-        SimpleDataGrapherObject.view.createButtons('yes');
-        var saveButton = SimpleDataGrapherObject.view.elementId + '_save_CSV';
-        var prevFile = SimpleDataGrapherObject.view.elementId + '_prev_file';
-        var saveFlag = false;
-        $('#' + saveButton).click(function(){
-            saveFlag = true;
-        });
+  var headerContainer = document.getElementsByClassName('body-container')[0];
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:13]:
```suggestion
    var headerContainer = document.getElementsByClassName('body-container')[0];
```

File: app/views/csvfiles/new.html.erb [hunk 2]
```diff
@@ -10,27 +10,27 @@
...
+  SimpleDataGrapherObject = new SimpleDataGrapher('first');
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:14]:
```suggestion
    SimpleDataGrapherObject = new SimpleDataGrapher('first');
```

File: app/views/csvfiles/new.html.erb [hunk 3]
```diff
@@ -10,27 +10,27 @@
...
+  var value = '<%= current_user %>';
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:15]:
```suggestion
    var value = '<%= current_user %>';
```

File: app/views/csvfiles/new.html.erb [hunk 4]
```diff
@@ -10,27 +10,27 @@
...
+  <% if current_user %>
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:16]:
```suggestion
    <% if current_user %>
```

File: app/views/csvfiles/new.html.erb [hunk 5]
```diff
@@ -10,27 +10,27 @@
...
+    SimpleDataGrapherObject.view.createButtons('yes');
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:17]:
```suggestion
      SimpleDataGrapherObject.view.createButtons('yes');
```

File: app/views/csvfiles/new.html.erb [hunk 6]
```diff
@@ -10,27 +10,27 @@
...
+    var saveButton = SimpleDataGrapherObject.view.elementId + '_save_CSV';
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:18]:
```suggestion
      var saveButton = SimpleDataGrapherObject.view.elementId + '_save_CSV';
```

File: app/views/csvfiles/new.html.erb [hunk 7]
```diff
@@ -10,27 +10,27 @@
...
+    var prevFile = SimpleDataGrapherObject.view.elementId + '_prev_file';
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:19]:
```suggestion
      var prevFile = SimpleDataGrapherObject.view.elementId + '_prev_file';
```

File: app/views/csvfiles/new.html.erb [hunk 8]
```diff
@@ -10,27 +10,27 @@
...
+    var saveFlag = false;
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:20]:
```suggestion
      var saveFlag = false;
```

File: app/views/csvfiles/new.html.erb [hunk 9]
```diff
@@ -10,27 +10,27 @@
...
+    $('#' + saveButton).click(function(){
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:21]:
```suggestion
      $('#' + saveButton).click(function(){
```

File: app/views/csvfiles/new.html.erb [hunk 10]
```diff
@@ -10,27 +10,27 @@
...
+      saveFlag = true;
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:22]:
```suggestion
        saveFlag = true;
```

File: app/views/csvfiles/new.html.erb [hunk 11]
```diff
@@ -10,27 +10,27 @@
...
   $('#' + SimpleDataGrapherObject.view.plotGraphId).click(function(){
     if (saveFlag){
-      var arr = {};
-      arr['completeCsvMatrix'] = SimpleDataGrapherObject.view.csvParser.completeCsvMatrix;
-      arr['csvHeaders'] = SimpleDataGrapherObject.view.csvParser.csvHeaders;
-      arr['csvSampleData'] = SimpleDataGrapherObject.view.csvParser.csvSampleData;
-      arr['csvValidForYAxis'] = SimpleDataGrapherObject.view.csvParser.csvValidForYAxis;
-      arr['completeCsvMatrixTranspose'] = SimpleDataGrapherObject.view.csvParser.completeCsvMatrixTranspose;
-      let csvStringMatrix = SimpleDataGrapherObject.view.csvParser.completeCsvMatrixTranspose;
-      let csvStringForDownload = encodeURI('data:text/csv;charset=utf-8,' + csvStringMatrix.map(e => e.join(',')).join('\n'));
```
Comment (TildaDares) [inline app/views/csvfiles/new.html.erb:33]: Hi @lanefoxwood, please revert the changes on lines 26-33. Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]
Feel free to just click the `Commit suggestion` button @lanefoxwood. Thanks!

File: app/views/csvfiles/new.html.erb (hunk 11 above)
Comment (lanefoxwood) [reply app/views/csvfiles/new.html.erb:33]: @TildaDares thanks, I will go ahead and revert those changes

File: app/views/csvfiles/new.html.erb [hunk 12]
```diff
@@ -14,23 +14,23 @@
...
+      SimpleDataGrapherObject.view.createButtons('yes');
+      var saveButton = SimpleDataGrapherObject.view.elementId + '_save_CSV';
+      var prevFile = SimpleDataGrapherObject.view.elementId + '_prev_file';
+      var saveFlag = false;
+      $('#' + saveButton).click(function(){
+        saveFlag = true;
+    });
   $('#' + SimpleDataGrapherObject.view.plotGraphId).click(function(){
     if (saveFlag){
-      var arr = {};
```
Comment (lanefoxwood) [inline app/views/csvfiles/new.html.erb:26]: I need to revert the changes made here lines 26-33

**Review by lanefoxwood — COMMENTED** [review]
I will try to do this ASAP!

File: app/views/csvfiles/new.html.erb (hunk 11 above)
Comment (lanefoxwood) [reply app/views/csvfiles/new.html.erb:33]: @TildaDares if you're busy and don't have time to help I understand, but I wanted to ask:
for reverting the changes I made to lines 26-33, what is the easiest way? I tried to edit the file in the pull request directly, but I don't think that's right. Would I have to put all of these commits into a new branch and make a new pull request or something? I'll keeping reading and trying to figure it out. Thanks for reading!

Comment (TildaDares) [reply app/views/csvfiles/new.html.erb:33]: Hi @lanefoxwood, I'm sorry if I wasn't clear enough but the original indentation on the file has the correct number of spaces, so we need to revert to lines 26-33 to how it was before. Lines 26-33 should be aligned with line 34. Thank you!!

Great job on figuring out how to add commits to an existing PR!

Comment (TildaDares) [reply app/views/csvfiles/new.html.erb:33]: Hi @lanefoxwood, do you need any help with this? Please let me know!

**Comment by TildaDares** [comment]
Resolved in #11133

**Comment by TildaDares** [comment]
Thank you for working on this @lanefoxwood!
